src/data/faq_de.json

```
@@ -46,6 +46,16 @@
                             "<b>National:</b> 0800 7540002",
                             "<b>International:</b> +49 30 498 75402"
                         ]
+                    },
+                    {
+                        "title": "Wie erstelle ich ein Fehler-Protokoll?",
+                        "anchor": "error_log",
+                        "active": true,
+                        "textblock": [
+                            "Mit Version 2.2 bietet die Corona-Warn-App eine Funktion zur technischen Problem-Analyse. Das Fehler-Protokoll soll den Entwicklern helfen, schneller eine Ursachen-Analyse zu ermöglichen und Fehler effizienter zu beseitigen.<br><b>Wichtig:</b> Melden Sie sich mit einem Problem bitte <b>zuerst</b> bei der Hotline (national: 0800 7540001 bzw. international: +49 30 498 75401), um zu prüfen, ob es ggf. schon eine Lösung dafür gibt. Ein Fehler-Protokoll wird nur benötigt, wenn das CWA-Team danach fragt.",
+                            "<ol><li>In einem solchen Fall öffnen Sie bitte ein Ticket auf GitHub und beschreiben Sie das Problem-Verhalten.</li><li>Starten Sie die Aufzeichnung in der CWA (---LINK oder Screenshot hier: Wie starte ich die Aufzeichnung?---)<br>- Versuchen Sie, das Verhalten zu reproduzieren bzw. die Fehlermeldung zu erzeugen.<br>- Ist das gelungen, senden Sie das technische Protokoll an den CWA-Server. Die Daten werden dabei anonymisiert und verschlüsselt. (Bei eigenem Interesse können Sie den Report auch lokal auf Ihrem Smartphone abspeichern.)<br>- Als Rückmeldung erhalten Sie eine eindeutige Kennung, die Sie auch in der Liste aller IDs finden <br>- Stoppen Sie das Fehler-Protokoll.</li><li>Aktualisieren Sie das GitHub-Ticket mit Ihrer eindeutigen Kennung vom Server, damit das aufgezeichnete Protokoll Ihrem Ticket zugeordnet werden kann.</li></ol>",
+                            "Hinweise:<ul><li>Die technischen Protokolle werden nach 14 Tagen automatisch gelöscht.</li><li>Protokolle, die unaufgefordert an den Server übertragen werden, können durch das CWA-Team nicht analysiert werden.</li><li>Für die Fehler-Protokolle gibt es keinen Rückkanal in die Corona-Warn-App. Das technische Team der CWA kann nur über GitHub zu einem Ticket kommunizieren.</li></ul>"

```

Some more comments on the text (align with app):

```suggestion
                            "Hinweise:<ul><li>Die Fehlerberichte werden nach 14 Tagen automatisch gelöscht.</li><li>Berichte, die unaufgefordert an den Server übertragen werden, können durch das CWA-Team nicht analysiert werden.</li><li>Für die Fehlerberichte gibt es keinen Rückkanal in die Corona-Warn-App. Das technische Team der CWA kann nur über GitHub zu einem Ticket kommunizieren.</li></ul>"
```

---

Indeed, aligned with App / UA ... excellent!